examples/client/Locomotion/src/pages/DevPage/index.tsx

```diff
@@ -13,7 +13,7 @@ import { InputContainer, Label } from './styles';
 
 const DevSettingPage = () => {
   const [operationId, setOperationId] = useState(Config.OPERATION_ID);
-  const [serverHost, setServerHost] = useState(Config.SERVER_HOST);
+  const [serverUrl, setServerUrl] = useState(Config.SERVER_HOST);
```

Comment:
why did you change from host to url?

Reply:
because the setting in `AppSettings` is called `serverUrl`, not `serverHost`, that was a typo bug